index.d.ts

```diff
@@ -1,5 +1,4 @@
 /// <reference types="node" />
-import type File = require("vinyl");
```

Comment:
Instead of moving the `require('vinyl')` down to the `ReplacerContext`, just remove the `type`-keyword here.

Reply:
Done ;)